utils/environment.go

```diff
@@ -52,36 +55,42 @@ func (e *environment) SetTimezone(timezone *time.Location) {
 	}
 }
 
+func (e *environment) Languages() LanguageList { return e.languages }
+
 //------------------------------------------------------------------------------------------
 // JSON Encoding / Decoding
 //------------------------------------------------------------------------------------------
 
 type envEnvelope struct {
-	DateFormat DateFormat `json:"date_format"`
-	TimeFormat TimeFormat `json:"time_format"`
-	Timezone   string     `json:"timezone"`
+	DateFormat DateFormat   `json:"date_format"`
+	TimeFormat TimeFormat   `json:"time_format"`
```

Comment:
Forgot about these guys, which we were going to have pass into the flow runner anyways, so yay for adding env!